apps/aehttp/src/sc_ws_handler.erl

```diff
@@ -246,6 +246,7 @@ handler_init_error(Err, Handler) ->
                     {channel_reserve_too_low      , value_too_low},
                     {push_amount_too_low          , value_too_low},
                     {lock_period_too_low          , value_too_low},
+                    {limit_exceeded               , limit_exceeded},
```

Comment:
Which limit? The error message should be more precise.

Reply:
That's partly a matter of strategy: do we have many detailed error types, or more general, perhaps complemented with more detailed info?

Reply:
So far most error messages are only used in a single case, so I expected the first option. But I don't have a preference.